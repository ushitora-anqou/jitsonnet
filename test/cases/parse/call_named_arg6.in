std.x(a, b=c)
