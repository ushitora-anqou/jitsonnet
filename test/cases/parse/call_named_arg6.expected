{ Syntax.v =
  (Syntax.Call
     ({ Syntax.v =
        (Syntax.Select
           ({ Syntax.v = (Syntax.Var "std");
              loc =
              { Syntax.fname = "test/cases/parse/call_named_arg6.in";
                ran =
                (Some { Syntax.startpos = { Syntax.line = 1; column = 0 };
                        endpos = { Syntax.line = 1; column = 3 } })
                }
              },
            "x"));
        loc =
        { Syntax.fname = "test/cases/parse/call_named_arg6.in";
          ran =
          (Some { Syntax.startpos = { Syntax.line = 1; column = 0 };
                  endpos = { Syntax.line = 1; column = 5 } })
          }
        },
      ([{ Syntax.v = (Syntax.Var "a");
          loc =
          { Syntax.fname = "test/cases/parse/call_named_arg6.in";
            ran =
            (Some { Syntax.startpos = { Syntax.line = 1; column = 6 };
                    endpos = { Syntax.line = 1; column = 7 } })
            }
          }
         ],
       [("b",
         { Syntax.v = (Syntax.Var "c");
           loc =
           { Syntax.fname = "test/cases/parse/call_named_arg6.in";
             ran =
             (Some { Syntax.startpos = { Syntax.line = 1; column = 11 };
                     endpos = { Syntax.line = 1; column = 12 } })
             }
           })
         ]),
      false));
  loc =
  { Syntax.fname = "test/cases/parse/call_named_arg6.in";
    ran =
    (Some { Syntax.startpos = { Syntax.line = 1; column = 0 };
            endpos = { Syntax.line = 1; column = 13 } })
    }
  }